public/app/features/correlations/Forms/QueryEditorField.tsx

```diff
@@ -13,6 +24,19 @@ interface Props {
 }
 
 export const QueryEditorField = ({ dsUid, invalid, error, name }: Props) => {
+  const [isValidQuery, setIsValidQuery] = useState<boolean | undefined>(undefined);
+  const theme = useTheme2();
+
+  const runner = useMemo(() => createQueryRunner(), []);
+
+  const styles = useMemo(() => {
+    return {
+      valid: css`
+        color: ${theme.colors.success.text};
+      `,
+    };
+  }, [theme]);
+
```

Comment:
what about using `usingStyles2` instead? something like

```ts
function getValidStyle(theme: GrafanaTheme2) {
  return {
    valid: css`
      color: ${theme.colors.success.text};
    `,
  };
}

export const QueryEditorField = ({ dsUid, invalid, error, name }: Props) => {
  const [isValidQuery, setIsValidQuery] = useState<boolean | undefined>(undefined);
  const styles = useStyles2(getValidStyle);
  // ...
```

Reply:
In order to make the use of our style theme more consistent I'll move it as you suggested.